acceptance/api/v1/service_list_test.go

```diff
@@ -0,0 +1,174 @@
+package v1_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/epinio/epinio/acceptance/helpers/catalog"
+	"github.com/epinio/epinio/acceptance/helpers/proc"
+	v1 "github.com/epinio/epinio/internal/api/v1"
+	"github.com/epinio/epinio/internal/services"
+	"github.com/epinio/epinio/pkg/api/core/v1/models"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	helmapiv1 "github.com/k3s-io/helm-controller/pkg/apis/helm.cattle.io/v1"
+)
+
+var _ = Describe("ServiceList Endpoint", func() {
+	var namespace string
+	var catalogService models.CatalogService
+
+	BeforeEach(func() {
+		namespace = catalog.NewNamespaceName()
+		env.SetupAndTargetNamespace(namespace)
+
+		catalogService = models.CatalogService{
+			Name:      catalog.NewCatalogServiceName(),
+			HelmChart: "nginx",
+			HelmRepo: models.HelmRepo{
+				Name: "",
+				URL:  "https://charts.bitnami.com/bitnami",
+			},
+			Values: "{'service': {'type': 'ClusterIP'}}",
+		}
+		createCatalogService(catalogService)
+	})
+
+	AfterEach(func() {
+		deleteCatalogService(catalogService.Name)
+		env.DeleteNamespace(namespace)
+	})
+
+	When("helmchart list is empty", func() {
+		It("returns a 404", func() {
```

Comment:
I think it should be HTTP 200 with empty list (see here too: https://stackoverflow.com/questions/13366730/proper-rest-response-for-empty-table)

Reply:
Yes, absolutely. It was my understanding that we were not doing like this, but it was not about collections, only for specific resources. :+1: